sdk/samples/iot/sample_sas_utility.c

```diff
@@ -28,15 +28,15 @@ az_result sample_base64_decode(az_span base64_encoded, az_span in_span, az_span*
   b64_decoder = BIO_new(BIO_f_base64());
   if (b64_decoder == NULL)
   {
-    return AZ_ERROR_OUT_OF_MEMORY;
+    return AZ_ERROR_HTTP_ADAPTER_OUT_OF_MEMORY;
```

Comment:
see comment: https://github.com/Azure/azure-sdk-for-c/issues/932#issuecomment-670062828